packages/back-end/src/services/datasource.ts

```diff
@@ -139,3 +178,42 @@ export async function testQuery(
     };
   }
 }
+
+export async function createInitialInformationSchema(
```

Comment:
I really don't like this naming convention, but I can't go with `createInformationSchema` as I have reserved that for the `InformationSchemaModel`'s create method. 

Would love suggestions here.

Reply:
Maybe `initializeDatasourceInformationSchema` ?